app/tests/test_collections_model.py

```diff
@@ -0,0 +1,500 @@
+import logging
+from datetime import datetime
+
+from django.conf import settings
+from django.test import TestCase
+
+from stac_api.models import Item
+from stac_api.utils import utc_aware
+
+import tests.database as db
+
+logger = logging.getLogger(__name__)
+
+API_BASE = settings.API_BASE
+
+
+class CollectionsModelTestCase(TestCase):
+
+    y100 = utc_aware(datetime.strptime('0100-01-01T00:00:00Z', '%Y-%m-%dT%H:%M:%SZ'))
+    y9000 = utc_aware(datetime.strptime('9000-01-01T00:00:00Z', '%Y-%m-%dT%H:%M:%SZ'))
+    y200 = utc_aware(datetime.strptime('0200-01-01T00:00:00Z', '%Y-%m-%dT%H:%M:%SZ'))
+    y8000 = utc_aware(datetime.strptime('8000-01-01T00:00:00Z', '%Y-%m-%dT%H:%M:%SZ'))
+
+    def setUp(self):
+        self.collection = db.create_collection('collection-1')
+
+    def add_range_item(self, start, end, name):
+        item = Item.objects.create(
```

Comment:
When manually creating object, we should always then call `.full_clean()` method that runs validation

Reply:
done